src/libponyc/CMakeLists.txt

```diff
@@ -125,7 +125,7 @@ elseif(${CMAKE_HOST_SYSTEM_NAME} MATCHES "Darwin")
     # on macos we dont have no static libc++
     # so when we want to use this lib
     # we need to additionally link libc++ and libz (for llvm)
-    file(GLOB_RECURSE LLVM_OBJS "${PROJECT_SOURCE_DIR}/../../build/build_libs/llvm/src/llvm//lib/libLLVM*.a")
```

Comment:
why was this change needed? i'm curious.

Reply:
The old folder was the folder where the llvm libs have been built in, before being copied to the `build/libs` dir. So this was working whenever the libs have been freshly built in CI. When the libs stuff has been loaded from the CI cache, only the copied stuff has been lying around.

It is safe to change it to this folder as a fresh build populates it with the right libs and the cache keeps them.

Reply:
gotcha